sdks/java/core/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
I don't know if this fits in this PR. It may be valuable, but should have broader discussion.